utils/oscap.8

```diff
@@ -303,6 +318,11 @@ Select a particular profile from XCCDF document.
 Allow download of remote OVAL content referenced from XCCDF by check-content-ref/@href.
 .RE
 .TP
+\fB\-\-local-files DIRECTORY\fR
+.RS
+Instead of downloading remote data stream components from the network, use a data stream component stored locally in a file. In place of the remote data stream component OpenSCAP will attempt to use a file whose file name is equal to @name attribute of the uri element within the catalog element within the component-ref element in the data stream if such file exists.
```

Comment:
```suggestion
Instead of downloading remote data stream components from the network, use data stream component stored locally in a file. In place of the remote data stream component OpenSCAP will attempt to use a file whose file name is equal to @name attribute of the uri element within the catalog element within the component-ref element in the data stream if such file exists.
```